server/src/test/java/io/crate/expression/scalar/HasSchemaPrivilegeFunctionTest.java

```diff
@@ -22,28 +22,32 @@
 package io.crate.expression.scalar;
 
 import static io.crate.testing.Asserts.isNotSameInstance;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import java.util.EnumSet;
+import java.util.List;
 import java.util.Set;
 
 import org.junit.Before;
 import org.junit.Test;
 
+import io.crate.exceptions.MissingPrivilegeException;
 import io.crate.metadata.information.InformationSchemaInfo;
 import io.crate.metadata.pgcatalog.OidHash;
 import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
-import io.crate.testing.Asserts;
 import io.crate.testing.SqlExpressions;
 import io.crate.user.Privilege;
 import io.crate.user.StubUserManager;
 import io.crate.user.User;
 
 public class HasSchemaPrivilegeFunctionTest extends ScalarTestCase {
 
-    private static User TEST_USER = User.of("test");
+    private static final User TEST_USER = User.of("test");
+    private static final User TEST_SUPER_USER = User.of("testSuperUser", EnumSet.of(User.Role.SUPERUSER));
```

Comment:
Why not use `User.CRATE_USER` instead everywhere? Currently this is the only user with superuser privileges.

Reply:
Since we're relaxing the check, it make sense to create a user with the `AL` privileges.